demo/rmm_plugin/rmm_singlegpu.py

```diff
@@ -0,0 +1,14 @@
+import xgboost as xgb
+import rmm
+from sklearn.datasets import make_classification
+
+# Initialize RMM pool allocator
+rmm.reinitialize(pool_allocator=True)
```

Comment:
What happen if I call this after DMatrix construction?

Reply:
The DMatrix will remain in the GPU memory, and a new memory pool will be allocated. Depending on the size of GPU memory and the memory pool, you may get an OOM error.